pathod/language/__init__.py

```diff
@@ -26,7 +26,7 @@ def parse_pathod(s, use_http2=False):
         May raise ParseException
     """
     try:
-        s = s.decode("ascii")
+        s.encode("ascii")
```

Comment:
We're only checking whether the string is valid ASCII here, so `encode` would be better since it works on both Python versions. Right?